Highlight required features (#247)
jenkinsci/jep: jep/221/README.adoc

== Abstract

This JEP introduces a new system which enables Jenkins plugin maintainers to add Continuous Delivery (CD) to their projects.

== Specification

=== Features

> * *Trusted CI/CD*:

Reviewer: beware that this might be construed more specifically as a particular server

Author: @jglick  
Can you suggest change?

    Deployments are produced from a single source of truth, which is managed by the Jenkins project.
* *Opt-in*:
    Plugin maintainers can choose to use or not this service.
    Those who choose not to use it will see no change to their engineering process.
* *Easy to use*:

>     Turning on this functionality is easy, requiring little to no additional configuration.

Reviewer: definitely some configuration, since it is opt-in

* *Source control driven*:

>     Depoloyments are triggered by changes to the source repository and are otherwise no-touch for users.

Reviewer: typo

>     Depoloyments are triggered by changes to the source repository and are otherwise no-touch for users.

Author: ```suggestion
    Deployments are triggered by changes to the source repository and are otherwise no-touch for users.
```

* *Assumes usage of current CI tools*:

>     For simplicity and stablity this system assumes plugin already uses `buildPlugin` and `incrementals`.

Reviewer: `incrementals` is not a code word. Can say _Incrementals_ or JEP-305.

>     For simplicity and stablity this system assumes plugin already uses `buildPlugin` and `incrementals`.

Author: ```suggestion
    For simplicity and stablity this system assumes plugin already uses `buildPlugin` and
    link:https://github.com/jenkinsci/jep/blob/master/jep/305/README.adoc[JEP-305].
```

[WARNING]
====
*(bitwiseman)* 
There's been a bunch of information in 
link:https://github.com/jenkinsci/jep/pull/244#discussion_r293534268[PR-244#discussion_r293534268] 
and
link:https://github.com/jenkinsci/jep/pull/244#issuecomment-510611838[PR-244#issuecomment-510611838]

> including refernce to 

Reviewer: typo

> including refernce to 

Author: @jglick  
```suggestion
including reference to 
```

[WARNING]
====
*(bitwiseman)* 

> There's a long thread here: https://github.com/jenkinsci/jep/pull/244#discussion_r293534268 that needs to be integrated into this JEP.

Author: Fixed link to thread with @jglick around design alternatives.
